services/app-api/src/handlers/postgres_migrate.ts

```diff
@@ -79,51 +109,68 @@ export const main: APIGatewayProxyHandler = async () => {
             const command = new CreateDBClusterSnapshotCommand(params)
             await rds.send(command)
         } catch (err) {
-            console.error(err)
-            return {
-                statusCode: 400,
-                body: JSON.stringify({
-                    code: 'DB_SNAPSHOT_FAILED',
-                    message:
-                        'Could not create a snapshot of the DB before migration: ' +
-                        err,
-                }),
-                headers: {
-                    'Access-Control-Allow-Origin': '*',
-                    'Access-Control-Allow-Credentials': true,
-                },
-            }
+            const errMsg = `Could not take RDS snapshot before migrating: ${err}`
+            recordException(
+                errMsg,
+                serviceName,
+                'CreateDBClusterSnapshotCommand'
+            )
+            return fmtMigrateError(errMsg)
         }
     }
 
     // run the data migration. this will run any data changes to the protobufs stored in postgres
     try {
-        execSync(
-            `${process.execPath} /opt/nodejs/protoMigrator/migrate_protos.js db '/opt/nodejs/protoMigrator/healthPlanFormDataMigrations'`,
+        const connectTimeout = process.env.CONNECT_TIMEOUT ?? '60'
+        const migrateProtosResult = spawnSync(
```

Comment:
should this be `await spawnSync`

Reply:
`spawnSync` doesn't return until the process returns, so it's not async like other types of `spawn` would be. I double checked though and we don't `await` it anywhere in our code base and my editor suggests removing it when you try to `await` it.